internal/engine/testdata/config_full.yaml

```diff
@@ -25,18 +25,23 @@ availableLanguages:
 collections:
   - id: adres
     search:
-      displayNameTemplate: "{straatnaam} {huisnummer}{huisletter}, {postcode} {woonplaats}"
+      displayNameTemplate: "{{straatnaam}} {{huisnummer}}{{huisletter}}, {{postcode}} {{woonplaats}}"
```

Comment:
Voor zowel displayNameTemplate als suggestTemplates geldt dat het Go text/template is. Dus overal `{{ .field }}` gebruiken.

Reply:
fixed 😔